compiler/z/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -9214,7 +9219,8 @@
                         S390OpProp_DoubleFP |
                         S390OpProp_UsesRegPairForSource |
                         S390OpProp_UsesRegPairForTarget |
-                        S390OpProp_SetsOperand1
```

Comment:
As this instruction is in form `LEXTR Rt,Rs,M3,M4`, Do we need `UsesM4` here as well?